docs/reference/webhook-notifications.md

```diff
@@ -0,0 +1,274 @@
+# Webhook Notifications
+
+Semaphore has webhook based notifications that are delivered on the success or
+failure of a pipeline. When criteria for notification are met, Semaphore will
+send a HTTP POST payload to the webhook's configured URL. You can use them,
+for example, to implement alerts through [Hubot](https://github.com/hubotio/hubot),
+or keep track of your projects on a company dashboard.
+
+## Setting up webhook notifications for a project
+
+Use your Endpoint URL to set up a notification on Semaphore,
+with the following command:
+
+``` bash
+$ sem create notification [name] \
+    --projects [project_name] \
+    --webhook-endpoint [webhook-endpoint]
+```
+
+For example, if you have a project called `web` and you want to get a webhook
+notification on every finished pipeline on the `master` branch, use the
+following command:
+
+``` bash
+$ sem create notification master-pipelines \
+    --projects web \
+    --branches master \
+    --webhook-endpoint [webhook-endpoint]
+```
+
+## Setting up webhook notifications for multiple projects
+
+When creating a notification, you can specify multiple projects as source,
+of your notifications.
+
+For example, if your team manages three projects named `web`, `cli` and `api`
+and you want to get notified for every finished pipeline on the master branch,
+use the following command:
+
+``` bash
+$ sem create notifications teamA-notifications \
+    --projects "web,cli,api" \
+    --branches "master" \
+    --webhook-endpoint [webhook-endpoint]
+```
+
+## Filtering by project, branch and pipeline names
+
+When creating a notification, you can specify a filter for project, branch and
+pipeline names.
+
+For example to send notifications for the `master` and `staging` branches use
+the following:
+
+``` bash
+$ sem create notifications example \
+    --branches "master,staging" \
+    --webhook-endpoint [webhook-endpoint] \
+```
+
+A filter can either be a regex match or a direct match. Specifying a filter with
+forward slashes (e.g. `/hotfix-.*/`) defines a regex match, while a filter without
+forward slashes (e.g. `master`) defines a direct match.
+
+``` bash
+$ sem create notifications example \
+    --branches "master,/hotfix\/.*/" \
+    --webhook-endpoint [webhook-endpoint] \
+```
+
+Matching can be specified for project and pipeline names as well. For example,
+if you want to get notified about every notification on a project that matches
```

Comment:
Please reword it to avoid 'notified about every notification' - I can help but we'd need to sync first.